tests/core/schemas/test_ase.py

```diff
@@ -345,6 +347,25 @@ def test_summarize_ideal_gas_thermo(tmp_path, monkeypatch):
         _summarize_ideal_gas_thermo(igt, charge_and_multiplicity=[0, 1])
 
 
+def test_summarize_harmonic_thermo(tmp_path, monkeypatch):
+    monkeypatch.chdir(tmp_path)
+
+    # Make sure metadata is made
+    ht = HarmonicThermo([0.34])
+    results = _summarize_harmonic_thermo(ht)
+    assert results["parameters_thermo"]["vib_energies"] == [0.34]
+    assert results["parameters_thermo"]["vib_freqs"] == [0.34 / invcm]
+    assert results["results"]["energy"] == 0
+    assert_close(results["results"]["helmholtz_energy"], 0.16999995401497991, rtol=1e-5)
+    assert_close(results["results"]["internal_energy"], 0.1700006085385999, rtol=1e-5)
+    assert_close(results["results"]["entropy"], 2.1952829783392438e-09, rtol=1e-5)
+    assert_close(results["results"]["zpe"], 0.17, rtol=1e-5)
+
+    # test document can be jsanitized and decoded
+    d = jsanitize(results, strict=True, enum_values=True)
+    MontyDecoder().process_decoded(d)
```

Comment:
This is great. Could you also test the "parent" `summarize_harmonic_thermo` function once so we make sure it's working as expected?

Reply:
I have added an additional unit test for `summarize_vib_and_thermo` which tests for both the `_summarize_harmonic_thermo` and `_summarize_ideal_gas_thermo` situation.